modules/tinymce/src/plugins/help/main/ts/ui/PluginsTab.ts

```diff
@@ -56,7 +56,7 @@ const tab = (editor: Editor): Dialog.TabSpec & { name: string } => {
   }, (x) => {
     // We know this plugin, so use our stored details.
     const name = x.type === PluginUrls.PluginType.Premium ? `${x.name}*` : x.name;
-    const html = makeLink({ name, url: `https://www.tiny.cloud/docs/tinymce/7/${x.slug}/` });
+    const html = makeLink({ name, url: `https://www.tiny.cloud/docs/tinymce/8/${x.slug}/` });
```

Comment:
Can you please find where we documented changes for a new major version @ltrouton might know?) and add this to it